lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp

```diff
@@ -7,6 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/BuiltinTypes.h"
```

Comment:
Are you sure that these are needed? Or, you just missed removing them?

Reply:
Auto-include crept back in it seems 😓